assets/src/web-stories-block/block/index.js

```diff
@@ -30,23 +25,15 @@ import metadata from './block';
 import { BLOCK_TYPE_LATEST_STORIES, GRID_VIEW_TYPE } from './constants';
 import transforms from './transforms';
 
-const { name, category, attributes } = metadata;
+const { name, category, attributes, supports, title, description, keywords } =
+  metadata;
 
 const settings = {
-  title: __('Web Stories', 'web-stories'),
-  description: __('Embed Web Stories.', 'web-stories'),
+  title,
+  description,
   category,
   icon,
-  keywords: [
-    /* translators: block keyword. */
-    __('embed', 'web-stories'),
-    /* translators: block keyword. */
-    __('web stories', 'web-stories'),
-    /* translators: block keyword. */
-    __('story', 'web-stories'),
-    /* translators: block keyword. */
-    __('stories', 'web-stories'),
-  ],
+  keywords,
```

Comment:
This change needs to be undone, otherwise these strings won't be translatable on the client.

For that to work, `registerBlockTypeFromMetadata` (WP 5.8+) would need to be used.

See https://github.com/WordPress/gutenberg/pull/30293 and https://github.com/WordPress/gutenberg/blob/722cd5c3b7ae888ac53a6ce6c69b3a0a9e4e9276/docs/reference-guides/block-api/block-metadata.md

Reply:
Reverted now.